code/_helpers/maths.dm

```diff
@@ -2,7 +2,7 @@
 #define RAND_F(LOW, HIGH) (rand()*(HIGH-LOW) + LOW)
 #define ceil(x) (-round(-(x)))
 #define CEILING(x, y) ( -round(-(x) / (y)) * (y) )
-#define NEAR_INFINITY	9999999999	//Just a really big number
+#define NEAR_INFINITY	1000000000	//Just a really big number
```

Comment:
I think it is better to do the value in the style of this #define INFINITY	1e31

Reply:
uhm, this is the largest integer that byond can support.